### DISCUSSION TITLE
Standalone go2rtc

### DISCUSSION THREAD

**Discussion by fredrik-tuomas-lv** [blakeblackshear/frigate#8065] [General]
Is there any downside of using a fully standalone go2rtc in a separate docker container? It would be easier to upgrade and tweak separate from Frigate.

I tried to search but did not find an answer to this question, maybe my search-fu is bad.

**Comment by NickM-27** [comment]
yes, you can't use the live views feature or the birdseye restream feature.

Frigate go2rtc is fully featured and can be upgraded by just downloading the binary to the config folder

**Reply by crossan007** [reply]
For reference, the documentation for that process is here: https://docs.frigate.video/configuration/advanced#custom-go2rtc-version

**Comment by crossan007** [comment]
It would also be nice to be able to run it in a separate container for sake of logging

`docker compose logs --follow --tail=100` is rather chaotic with so many "services" running within the default `ghcr.io/blakeblackshear/frigate:stable`  frigate container

**Reply by NickM-27** [reply]
You can already get the separated logs from the webUI api

**Reply by crossan007** [reply]
True.

I'm just trying to investigate a specific log entry and I'm not sure who's logging it.

I'm not using this thread to seek assistance with this logged issue; only sharing it to illustrate that the web UI has a few blind spots about "who's logging what"

![image](https://github.com/user-attachments/assets/851b7d25-7fdc-42d8-b62b-987977163298)

```
[h264 @ 0x7f789c11d580] error while decoding MB 76 41, bytestream -6
```

**Reply by NickM-27** [reply]
That is on the frigate tab so it is coming from frigate. It has no tag so it is tagged as unknown. It is most likely opencv logging it
